### DISCUSSION TITLE
Need help placing link button on top of image

### DISCUSSION THREAD

**Discussion by hilarytsai** [mozilla/hubs#3998] [Spoke Questions] [answered]
Someone taught me a trick to hide an image associated with a link so that only the button shows through on top of another image.  As I recall, it involves changing the position coordinates so that the link is .1 less than the image you want to show.  (In my case, it's the x axis).  I also remember they told me it only works from a specific direction.

 I don't know exactly what the problem is here, but I cannot successfully get the image associated with the link button to be hidden and only show the button.  Right now, I have the entire link image in front of the intended image--but it's ugly, that's why I want to hide it.  I'd love if anyone could take a look and see if you can get only the button to show through.  If you look in my file, the link and target image are at the bottom.  It's Ashlin (link) and ashlinscreenshot.png.  The X axis is the one you want to move in this case.  No worries if you can't, I figured it can't be done; but I thought I'd put it out there to see if it's possible.

Thank you!!!
https://hubs.mozilla.com/scenes/o9zr9om

**Comment by johnshaughnessy** [comment, marked as answer]
Images in spoke can now be used as a link directly so that you don't have to do the trick where you hide the link element slightly behind an image. Fill in the `Link Href` field and your image will act like a link but appear like the image.

![image](https://user-images.githubusercontent.com/4072106/110978391-2ea5be80-8318-11eb-8c3f-8fa9b8542f77.png)
